package.json

```diff
@@ -37,58 +37,58 @@
     "postpublish": "git push --follow-tags origin HEAD"
   },
   "dependencies": {
-    "clsx": "^1.1.0"
+    "clsx": "^1.1.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.10.1",
-    "@babel/plugin-transform-runtime": "^7.10.1",
-    "@babel/preset-env": "^7.10.1",
-    "@babel/runtime": "^7.10.1",
-    "@popperjs/core": "^2.4.2",
+    "@babel/core": "^7.11.0",
+    "@babel/plugin-transform-runtime": "^7.11.0",
+    "@babel/preset-env": "^7.11.0",
+    "@babel/runtime": "^7.11.0",
+    "@popperjs/core": "^2.4.4",
     "@storybook/react": "^5.3.19",
-    "@testing-library/react": "^10.0.4",
+    "@testing-library/react": "^10.4.7",
     "@types/enzyme": "^3.10.5",
     "@types/enzyme-adapter-react-16": "^1.0.6",
     "@types/faker": "^4.1.12",
     "@types/jest": "^25.2.3",
-    "@types/lodash": "^4.14.153",
-    "@types/node": "^14.0.5",
-    "@types/react": "^16.9.35",
+    "@types/lodash": "^4.14.158",
+    "@types/node": "^14.0.27",
+    "@types/react": "^16.9.44",
     "@types/react-dom": "^16.9.8",
-    "@types/react-select": "^3.0.13",
+    "@types/react-select": "^3.0.15",
     "@types/react-virtualized": "^9.21.10",
-    "@typescript-eslint/eslint-plugin": "^3.0.2",
-    "@typescript-eslint/parser": "^3.0.2",
+    "@typescript-eslint/eslint-plugin": "^3.7.1",
+    "@typescript-eslint/parser": "^3.7.1",
     "babel-loader": "^8.1.0",
-    "babel-plugin-optimize-clsx": "^2.6.0",
+    "babel-plugin-optimize-clsx": "^2.6.1",
     "clean-css": "^4.2.3",
     "core-js": "^3.6.5",
-    "css-loader": "^3.5.3",
+    "css-loader": "^3.6.0",
     "enzyme": "^3.11.0",
     "enzyme-adapter-react-16": "^1.15.2",
-    "eslint": "^7.1.0",
-    "eslint-plugin-jest": "^23.13.2",
+    "eslint": "^7.6.0",
+    "eslint-plugin-jest": "^23.20.0",
     "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-react": "^7.20.0",
-    "eslint-plugin-react-hooks": "^4.0.4",
+    "eslint-plugin-react": "^7.20.5",
+    "eslint-plugin-react-hooks": "^4.0.8",
     "eslint-plugin-sonarjs": "^0.5.0",
     "faker": "^4.1.0",
-    "jest": "^26.0.1",
-    "less": "^3.11.1",
-    "less-loader": "^6.1.0",
-    "lodash": "^4.17.15",
+    "jest": "^26.2.2",
+    "less": "^3.12.2",
+    "less-loader": "^6.2.0",
+    "lodash": "^4.17.19",
     "mini-css-extract-plugin": "^0.9.0",
     "react": "^16.13.1",
-    "react-contextmenu": "^2.13.0",
+    "react-contextmenu": "^2.14.0",
     "react-dnd": "^11.1.1",
     "react-dnd-html5-backend": "^11.1.1",
     "react-dom": "^16.13.1",
     "react-popper": "^2.2.3",
     "react-select": "^3.1.0",
     "react-virtualized": "^9.21.2",
-    "ts-jest": "^26.0.0",
+    "ts-jest": "^26.1.4",
     "ts-loader": "^7.0.5",
-    "typescript": "~3.9.3"
+    "typescript": "^3.9.7"
```

Comment:
I would have prefered `~` instead of `^`, but since the next version will be 4.0.0 we can leave it as is.